spec/spec_helper.rb

```diff
@@ -0,0 +1,19 @@
+require "logstash/devutils/rspec/spec_helper"
+require 'logstash/inputs/twitter'
+require 'twitter'
+
+class MockClient
+  def filter(options)
+    loop { yield }
+  end
+
+  alias_method :sample, :filter
+end
+
+def run_input_with(input, queue)
+  t = Thread.new(input, queue) do |_input, _queue|
+    _input.run(_queue)
+  end
+  sleep 0.1
```

Comment:
me neither, but at my understanding there is no way to just do for example 1 run as we need the run method to be an infinite loop, anyway....